app/src/main/java/com/andela/omokolataiwo/levelup/presenter/GithubProfilePresenter.java

```diff
@@ -21,36 +23,41 @@
      */
     private final MainContract.MainView mView;
 
+    /**
+     * Coordinator layout.
+     */
+    private final CoordinatorLayout coordinatorLayout;
+
     /**
      * GithubProfilePresenter constructor.
      *
-     * @param view Activity of the request.
+     * @param view              Activity of the request.
+     * @param coordinatorLayout coordinator layout
      */
-    public GithubProfilePresenter(MainContract.MainView view) {
+    public GithubProfilePresenter(MainContract.MainView view, CoordinatorLayout coordinatorLayout) {
         this.mView = view;
+        this.coordinatorLayout = coordinatorLayout;
     }
 
-
     @Override
     /**
      * Fetches Github users' profile for Github API
      */
     public void fetchData() {
-        RetrofitClientInstance
-                .getRetrofitClientInstance()
-                .getAllProfile()
+        RetrofitClientInstance.getRetrofitClientInstance().getAllProfile()
```

Comment:
@omokolataiwo 
I believe this `RetrofitClientInstance` is a `Singleton` class....awesome.
Can you try renaming it to something like this `RetrofitClient`
Then you can also rename the `getRetrofitClientInstance` method to `getInstance`....so you should be have something like this-- `RetrofitClient.getInstance().getAllProfile`
The reason here is because, for every standard `Singleton` class or object structure, it usually follows a particular pattern which is:
`TheClassName.getInstance()`
E.g. Calendar calendar = Calendar.getInstance(); // Java calendar api